packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -156,6 +166,32 @@ const DeleteOldBackupsFirst = ({ handler, handlerParam, value }) => (
   </ActionButton>
 )
 
+const RemoteProxyWarning = decorate([
+  addSubscriptions({
+    remotes: cb =>
+      subscribeRemotes(remotes => {
+        cb(keyBy(remotes, 'id'))
+      }),
+  }),
+  provideState({
+    computed: {
+      showWarning: (_, { id, proxyId, remotes = {} }) => {
+        const remote = remotes[id]
+        return (
+          proxyId != null && remote !== undefined && remote.proxy !== proxyId
```

Comment:
So a remote is globally bound to a proxy, right? Can you select a remote that is bound to a proxy without selecting that proxy first? Does it still work or do you need to unbind the remote first?

Reply:
Right, I don't handled this case :+1: